.expeditor/config.yml

```diff
@@ -48,40 +48,18 @@ github:
   # Which Github branches to build Omnibus releases from, and what versions
   # (as determined by the value in the VERSION file) those branches are responsible
   # for building.
-  release_branch:
-    - master:
-        version_constraint: 16*
-    - chef-15:
-        version_constraint: 15*
-    - chef-14:
-        version_constraint: 14*
+
+release_branches:
+  - master:
+      version_constraint: 16*
+  - chef-15:
+      version_constraint: 15*
+  - chef-14:
+      version_constraint: 14*
 
 changelog:
   rollup_header: Changes not yet released to stable
 
-# These actions are taken, in order they are specified, anytime a Pull Request is merged.
```

Comment:
Let's keep this comment so its inline with the new configuration.

Reply:
added this comment